assets/scss/6-components/info-drop/_info-drop.scss

```diff
@@ -0,0 +1,81 @@
+// Info drop (c-info-drop)
+//
+// Uses `<details>` and `<summary>` tags to create an information dropdown component. This includes fallbacks for IE and Edge, which currently don't support those tags. Edge has marked implementation for those as high-priority. {{isWide}}
+//
+// Markup: 6-components/info-drop/info-drop.html
+//
+// Styleguide 6.1.3
+
+$info-drop-size: px-to-rem(14px);
+
+.c-info-drop {
+  height: $info-drop-size;
+  position: relative;
+
+  &__inner {
+    position: absolute;
+  }
+
+  &__content {
+    max-height: 0;
+    transition: max-height 0.3s;
```

Comment:
I remember reading a while back that it's more efficient for the browser to animate `transform: translateY` than `height/max-height`. Probably doesn't matter all that much here.

Reply:
Translation animations certainly look better. I'll try that out. I know I had it at one point, but not sure what got in my way. (I probably animated it 10 different ways 😆 before I went with max, but I secretly hate animating with max-height.)

Reply:
urgg yeah so I tried it again and I remember now. If we want that slide-down look, it almost has to grow in height and the element that grows _with_ the open attribute to key off of is the ` c-info-drop__container`. I can `transform: translateY(someval)` but it'll basically only slide that container within, not the entirety of what you see in blue. The tricky layout piece of this whole component was that container and the fact that this needs to overlay the elements underneath it, not push down. The `c-info-drop__inner` might the potential to animate it's growing/shinking, but it's outside of the `open=""` specifier.

All that goes to say, if it degrades page perf too much, we can settle on no slide-down look. The text still fades in 🤷‍♀

Reply:
Yeah I don't think this is worth exerting too much effort on!